tests/spec/DeprecatedSpec.js

```diff
@@ -1,9 +1,9 @@
-describe("Deprecated Methods and Properties", function () {
+xdescribe("Deprecated Methods and Properties", function () {
 
 
-	describe("Deprecated Functions", function(done) {
```

Comment:
What was the done doing?

Reply:
In this case... Nothing. Which is why I removed it.

The version of jasmine that I installed gave errors that one of these functions didn't actually accept any arguments... I think it was the done callback in the `describe` test function argument. 

Usually it is intended as a callback for asynchronous tests to end.
https://jasmine.github.io/tutorials/async